web_console/capabilities_products-web-console.adoc

```diff
@@ -22,7 +22,8 @@ include::modules/pipelines-web-console.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* link:https://access.redhat.com/documentation/en-us/red_hat_openshift_pipelines/1.13/html/creating_cicd_pipelines/working-with-pipelines-web-console[Working with {pipelines-title} in the web console]
+* link:https://docs.openshift.com/pipelines/1.14/create/working-with-pipelines-web-console.html[Working with {pipelines-title} in the web console]
+* link:https://docs.openshift.com/pipelines/1.14/create/working-with-pipelines-web-console.html#op-console-statistics_working-with-pipelines-web-console[Pipeline extecution statistics in the web console]
```

Comment:
typo "extecution"

Reply:
Updated! Thanks for catching that @mramendi